### ISSUE TITLE
Itemban spam before logging in?

### ISSUE THREAD

**Issue by tysonstrange** [NyxStudios/TShock#1072]
Not sure if this is really an "issue", but players get spammed with the red "This item is banned, please remove it" messages, even before they've logged in.

Should/Could this have a player.IsLoggedIn check wrapped around it, since the IsItemBanned check is always going to return false on a player that's not logged in?

**Comment by Kromara** [comment]
Do you have SSC on? If so, that can do that, I believe. I have nearly every
equippable item restricted with item bans, so I'm not too certain anymore

**Comment by rautamiekka** [comment]
Shouldn't SSC force a totally separate inventory on the user regardless of what he had before connecting ?

**Comment by Enerdy** [comment]
No, it only forces a new inventory when they log in, so unless they log on before join, that won't be the case. Same for players before registering.

**Comment by tysonstrange** [comment]
Yes I'm using ssc.

I log in, it shows singleplayer inventory until you log in or if you use /logout.

you cant drop items until you're logged in though, so people cant duplicate or sneak in items, but you still get the spam

**Comment by tysonstrange** [comment]
Also... i dont think dyes are checked for item bans... You can hold/use them and get disabled but if you just put the dye on it seems to work fine.

**Comment by TeeriaLLC** [comment]
2015-09-11 16:22:31 - TSPlayer: INFO: Player Brady has been disabled for check ignores failed in OnSecondUpdate().
2015-09-11 16:22:36 - TSPlayer: INFO: Player Brady has been disabled for check ignores failed in OnSecondUpdate().
2015-09-11 16:22:41 - TSPlayer: INFO: Player Brady has been disabled for check ignores failed in OnSecondUpdate().
2015-09-11 16:22:46 - TSPlayer: INFO: Player Brady has been disabled for check ignores failed in OnSecondUpdate().
2015-09-11 16:22:51 - TSPlayer: INFO: Player Brady has been disabled for check ignores failed in OnSecondUpdate().
2015-09-11 16:22:56 - TSPlayer: INFO: Player Brady has been disabled for check ignores failed in OnSecondUpdate().
2015-09-11 16:23:01 - TSPlayer: INFO: Player Brady has been disabled for check ignores failed in OnSecondUpdate().
2015-09-11 16:23:06 - TSPlayer: INFO: Player Brady has been disabled for check ignores failed in OnSecondUpdate().
2015-09-11 16:23:11 - TSPlayer: INFO: Player Brady has been disabled for check ignores failed in OnSecondUpdate().
2015-09-11 16:23:16 - TSPlayer: INFO: Player Brady has been disabled for check ignores failed in OnSecondUpdate().
2015-09-11 16:23:21 - TSPlayer: INFO: Player Brady has been disabled for check ignores failed in OnSecondUpdate().
2015-09-11 16:23:26 - TSPlayer: INFO: Player Brady has been disabled for check ignores failed in OnSecondUpdate().
2015-09-11 16:23:31 - TSPlayer: INFO: Player Brady has been disabled for check ignores failed in OnSecondUpdate().
2015-09-11 16:23:36 - TSPlayer: INFO: Player Brady has been disabled for check ignores failed in OnSecondUpdate().

Basically that piece of crap, it's causing non-SSC servers to lag fricking badly and it's annoying.
